File: Java/Circumference.java
class Circumference {
	public static void main(String[] args) {
		float r=5.0F, c;
		MyLib m = new MyLib();
		c = 2 * m.pi() * r;
		System.out.println("The circumference is " + c);
	}
}

/*
	out is not an object of System class. Infact, it is an object
	of printstream class.
*/

File: CPP/Circumference.cpp
#include<iostream>
#include "MyLib.cpp"
using namespace std;
//using namespace lib2;
int main()
{
	float r, c;
	cout<<"Enter radius: ";
	cin>>r;
	lib1::mylib m1;
	c = 2 * m1.pi() * r;
	cout<<"The circumference using pi as 22/7 is "<<c<<endl;
	lib2::mylib m2;
	c = 2 * m2.pi() * r;
	cout<<"The circumference using pi as 355/113 is "<<c<<endl;
	return 0;
}
